### DISCUSSION TITLE
MNAF Senary Saloon - First Release

### DISCUSSION THREAD

**Discussion by Mihro** [Mihro/MNAF-Senary-Saloon#2] [Announcements]
# Download & Install
### Requirements
- Minecraft: Java Edition 1.16.5
- Optifine 1.16.5 HD_U_G7

## World + Resource Pack
1. Download the latest release below; `Assets > Source code (zip)`
2. Place the zip in your `saves/<worlds>` folder.  
   Eg. `.minecraft/saves/MNAF-Senary-Saloon-main.zip`
3. Open Minecraft and play the world. The resource pack should automatically apply.

## Standalone Resource Pack
If you're playing on multiplayer then others will need the resource pack.
1. Download `MNAF-Senary-Saloon-RP.zip` below.
2. Place the zip in your `resourcepacks/` folder.  
   Eg. `.minecraft/resourcepacks/MNAF-Senary-Saloon-RP.zip`
3. Enable the resource pack in Minecraft.

---

# How to Play
The most useful thing in the map is the "MNAF Handy-Dandy Book" which lists shortcuts to most of the features that are available.  
You can get one using the command `/function mnaf:help_book`. _You must be opped to use it._

### Wiki
Visit the [Senary Saloon Wiki](https://github.com/Mihro/MNAF-Senary-Saloon/wiki/How-to-Play) for more info

<hr /><em>This discussion was created from the release <a href='https://github.com/Mihro/MNAF-Senary-Saloon/releases/tag/v1.0.0'>MNAF Senary Saloon - First Release</a>.</em>

**Comment by uqbah20214** [comment]
the animatronics the skins dont work when i put them on i only have the head but the body is chanmeal for some reason

**Reply by uqbah20214** [reply]
its works

**Comment by uqbah20214** [comment]
![idk](https://user-images.githubusercontent.com/89732490/131249800-d59779e1-7baf-4b6d-a3d3-142735cf500d.png)

**Reply by tayvinb1** [reply]
do you have optifine on? if so are you on 1.16.5? if you are doing both of those then idk what the issue is
